src/gam.py

```diff
@@ -7432,7 +7434,7 @@ def doGetVaultMatterInfo():
   matterId = convertMatterNameToID(v, sys.argv[3])
   result = callGAPI(v.matters(), u'get', matterId=matterId, view=u'FULL')
   if u'matterPermissions' in result:
-    cd = buildGAPIObject(u'directory')
+    buildGAPIObject(u'directory')
```

Comment:
I think this was a mistake on my part, should have built cd here and then passed it to:

user_email = convertUserUIDtoEmailAddress(

further down so that we don't waste cycles on building directory API object every time through the loop